### ISSUE TITLE
UX Discussion: LOTR, What's on the Menu?, Orbis

### ISSUE THREAD

**Issue by ebeshero** [ebeshero/DHClass-Hub#741]
Choose one of the following digital archives to explore, and write a post in this thread discussing: 
1) how visual aids are used to convey information on the site, and how well (or badly) do these work? What seems unclear to you, and/or what seems to work really well? 
2) how effective is the  **user experience** (“UX”) in navigating at least one or two significant areas of the site? 
(Each of you should begin by writing a "UX review" of a site, and then read and respond to each other by looking at the sites others chose.) 

* [LOTR Project: Visualizing Tolkien’s Works on the Web](http://lotrproject.com/)
* [What’s on the menu?](http://menus.nypl.org/) (be sure to read the About page for the project goals)
* [ORBIS](http://orbis.stanford.edu/): The Stanford Geospatial Network Model of the Roman World

@ChinoyIndustries @smdunn921 @amberpeddicord @bobbyfunks @lmcneil7 @benjaminc2020 @Bryant-LettucePrime @biancamaginley @robftg @frabbitry @BMT45 @ebeshero

**Comment by ebeshero** [comment]
Just to be clear, we'll finish up the Traversing the Tree discussion tonight (M 1/13 by 11:59pm EST) and then launch this one for T and W this week.

**Comment by amberpeddicord** [comment]
I'll jumpstart this discussion! I chose to look at the "What's on the menu" site. 

For this project, the transcribing and markup is primarily run by the public. As is described on the 'About' page, the site features a tool that allows users to transcribe the recipes to code for data about ingredients, prices, locations, etc. Because of this, the User Experience is crucial to the project's success. 

And, I think they're successful at laying out each page in a way that is simple enough to navigate, but also relays adequate information. On the index page, they provide the user with links to necessary information about the project, and also links to the transcription tools in several places. On the 'Dishes' page, the dishes are clearly laid out by year, and this section is also easy to navigate. 

However, the pages for each dish seem to have a lot of unused/misused space. It isn't necessarily difficult to see necessary data about each meal, but the information is scattered. There isn't a logical order to things. 
That being said, I thought this project was a really interesting way to show how public libraries can use digital technologies for their archival methods--and how they can get the public involved in this! As someone who works as a social media/marketing intern for a public library, public involvement isn't always an easy thing to accomplish, so I found this project really interesting!

**Comment by benjaminc2020** [comment]
I reviewed the LOTR Project. On this site, the visual aids are generational family trees to display the lineage of each race of characters (elves, dwarves, orcs, hobbits, etc..) in the LOTR series. In my opinion, the display and outline of the trees are difficult to wrap your mind around. The trees are very extensive and can become overwhelming trying to follow without zooming out on the site. The info provided when clicking on a single name will lead you to a quick bio (if available) and link to another site to read more about the character. This is a pretty good feature to include, although I would have liked to have seen a quick image of the character in the pop-up bio when the character name is clicked, as well as what movie/book they were featured in. Those types of features would improve the UX because they would lessen the amount of the time the user has to navigate to figure something out. 

This site does really excel at providing data visualizations in pretty much every type of data you'd like to find out about the series, especially in the timelines and maps, although as a user I would still appreciate some more info in the quick pop-up rather than quickly suggesting I go to the Tolkien Gateway site to find out more. The Search, Navigation, and Highlight tabs in the top right do help simplify the family tree data set, especially when looking for a single race.

**Comment by lmcneil7** [comment]
I took a look at the ORBIS Project! From my first impression on the site, it's a little cluttered. Everything is necessary information for the map, but I think it would be more comfortable on the eyes if one of the sidebars were smaller or had its page. Speaking about the visual aids, each one is relevant to maps. The visual aid on the bottom gives us background information about Alexandria and the Roman World and provides us a key to the map. The visual aid on the left is a tool to calculate a route, network, or flow from different cities in the Roman World. The visual aid on the right allows you to highlight what you want to see on the map at any given moment. Each visual has a reason for existing and works well for the site as a whole.  

As I mentioned, there's a visual aid that I don't understand, which affects the UX because that means that people will be confused but have no idea how to figure it out. They should include something to describe what each visual aid does, even if it seems simple enough. Another approach would have an example page to walk their audience through each aid and how it can help them plan out possible trips. The colors work well together, but I think it would look better if there were more differences in the color instead of all red plus black and white. The information works well on one page, but it would be better if there were other pages to the site.

I didn't check out the other pages, but in other people's opinion on them, I think that ORBIS is similar as it provides a good bit of information creatively, but still has room for improvement.

**Comment by bobbyfunks** [comment]
I looked at the LOTR project.  I like the menu selections and the different perspectives that you can view the map with.  I had issues with the "Timelines" and "events" features on the map feature  and I am not sure if it is my computer or a bug  When I go to the "Timeline" tab it comes up but not in the maps section.  The different graphs indicating various information about the different races and is well done.  I generally enjoyed the site since I like maps and LOTR!

**Comment by smdunn921** [comment]
I also looked at the LOTR project. The site really hits you in the face as soon as you enter. There is a *lot* of information to have to try to figure out. It does seem especially cool and thorough information-wise, but again having to figure out what is going on seems like a bit of a challenge. 

The home page has a family tree sort of deal, but there is a section on the top right called Navigation, where you can click on each certain group and it will take you to that part of the extensive tree, as well as an option to highlight specific things within the entirety including men, hobbits, elves, and half-elves. 

Upon getting further into the site, there's more interactive maps, where you can click to see different settlements, bodies of water, roads, etc, which is a lot less daunting than what's on the homepage and therefore easier to figure out. The site also includes a timemap, which can again be color coded, this time by character, to show different findings. The statistics page has all kinds of graphs touching on statistics of middle earth, which are very in depth and come with a couple paragraphs of explanation underneath.

All in all, even though the index page hits you in the face, the rest of the site layout is pretty interesting, informative, and thorough. I think that this project is super fun to look at after your eyes adjust to it for the first minute you have to sit there and think "okay what am i looking at?"

**Comment by benjaminc2020** [comment]
> I'll jumpstart this discussion! I chose to look at the "What's on the menu" site.
> 
> For this project, the transcribing and markup is primarily run by the public. As is described on the 'About' page, the site features a tool that allows users to transcribe the recipes to code for data about ingredients, prices, locations, etc. Because of this, the User Experience is crucial to the project's success.
> 
> And, I think they're successful at laying out each page in a way that is simple enough to navigate, but also relays adequate information. On the index page, they provide the user with links to necessary information about the project, and also links to the transcription tools in several places. On the 'Dishes' page, the dishes are clearly laid out by year, and this section is also easy to navigate.
> 
> However, the pages for each dish seem to have a lot of unused/misused space. It isn't necessarily difficult to see necessary data about each meal, but the information is scattered. There isn't a logical order to things.
> That being said, I thought this project was a really interesting way to show how public libraries can use digital technologies for their archival methods--and how they can get the public involved in this! As someone who works as a social media/marketing intern for a public library, public involvement isn't always an easy thing to accomplish, so I found this project really interesting!

I agree with what you're saying about the layout and use of space with this website. I would go even further and say that the design quality overall could use a revamp. For the most part, there are just simple issues, like the font is too small, too much is being displayed "above the fold", and the information (especially the dishes page) needs more of a logical structure rather than being scattered. Overall the website does have a pretty interesting purpose considering the significance that time and date can have on food and menus.

**Comment by biancamaginley** [comment]
I took a look at the Lord of the Rings website. At first glance, I was confused about the layout, that didn't change much the longer I looked at it. Firstly, the navigation of the home page is inconvenient. A vertical scroll bar is simpler than a horizontal one. Secondly, there is a lot of white space on the home screen. The "family tree" takes up only the top portion, but the links on the side continue down for a considerable distance. That makes the website look rushed. I do like the fact that the menu at the top of the screen stays visible as you scroll down, makes easy access. The statistics page is a lot clearer of an experience, and doesn't have the same layout as the main page. It looks like it was created by two different people. The statistics page was attractive and the charts appeared to have the same theme, so it was a lot better to look at. 

The navigation of the home page is downright awful. The family tree looks more like it belongs on an app than a website. All in all, not a good first impression in terms of user experience. The statistics page, as I mentioned, was a much better experience. The layout made sense and it was friendly on a computer. One thing that was a little off was the text under the statistics. In some places it was even and well formatted, but in others there was more information on one side than the other, which took extra room. The site as a whole looks like it would be interesting and fun to use when either reading or watching the movies, just to keep track of the many characters and events. Some simple editing would make the UX better.

**Comment by biancamaginley** [comment]
BTW, I didn't mean to close the issue. My apologies.

**Comment by ebeshero** [comment]
@biancamaginley No worries--it's easy to do! Thank for re-opening it! :-)

**Comment by Bryant-LettucePrime** [comment]
I actually took a UX course last semester, so I feel undeservedly qualified to comment on the LOTR project's design.

Early in the course we outlined six (of many) useful measures of User Experience that I'll apply to this website:

1.) Visibility - Clarity of information
- The LOTR project's interactable elements are clearly labeled and easily identified. Most are buttons collected in one portion of the screen with translucent backgrounds and easily readable text.

2.) Mappings - Obvious connections of information
- This doesn't necessarily have to be complicated, and overall the way the site lays out relevant data and elements makes sense.  One might expect a JavaScript function to fire to change the background of the page to the relevant selected item while the navigation buttons at the top of the screen to remain the same, but every button loads a new page - which is unexpected to a snooty privileged child of the digital era, but just a stylistic issue.  My only complaint may be that, on the page linked in the above post, there's a wonderful family tree lifted right out of the _Silmarillion,_ and, like the _Silmarillion_ it's too hard to read.

~ ALSO ~

At least one link on the 'Store' page takes you to a Japanese website.  That your internet traffic is about to cross the pond is never communicated and I have no earthly idea why it's the case at all.

3.) Affordance/Constraints - Obvious communication of expected user input
- Bad affordance is when a door handle makes you think you should pull instead of push.  Because the design was not properly constrained, it offers the _possibility_ to use it incorrectly, the likelihood of user error increases.  This doesn't really seem to be an issue on this site because it just isn't rocket science to navigate.

Perhaps one minor quibble is that innocuous buttons such as 'Blog', 'Contact' and 'Share' share space with cool stuff like 'Maps', 'Timelines', and 'Statistics'.  Furthermore, the 'More' category has a menagerie of fun stuff in an extra menu that's unceremoniously crammed between the 'Store' and 'Contact' buttons.  Furthermore, I wonder about having three dedicated Social Media buttons right next to 'Share', especially when the links are at the bottom of almost every webpage anyway. Maybe just make them smaller and in some other part of the screen - pare down the options to only those relevant to the information for which we're accessing the site.

4.) Conceptual Model - The rule-set the user intuits that will govern their use of the site.
- Press button, see stuff.  It's simple.  Insulting, really.

5.) Feedback - Provides the user with an acknowledgement of their interaction with the site.
- THIS is the my biggest bugger.  Perhaps it's just my internet, but load times are bad.  All the programming under the head has likely hurt optimization and meant that some functionality is slow and cumbersome.  The interactive map is especially bad - and as an aside, some of it no longer works.  If one waits the eight million years for the 'Timelines' panel on the right-hand side of the screen to load, and they're greeted with an index of well curated, impressively designed, really quite stylish error messages.  (And that's when it loads at all.)

In short, I like the site, and I like Tolkien, but it's transparently an imperfect labor of love and not corporate Skinner-box with all imperfection sand-blasted off.  That's fine, charming even, but every now and then the stretched tech impedes functionality.
